tests/user_examples/extra_src/shared/general_continuum/general_continuum.h

```diff
@@ -41,5 +41,35 @@ class GeneralContinuum : public WeaklyCompressibleFluid
 
     virtual GeneralContinuum *ThisObjectPtr() override { return this; };
 };
+
+class PlasticContinuum : public GeneralContinuum
+{
+protected:
+    Real c_;  /*< cohesion  */
+    Real fai_;  /*< friction angle  */
+    Real psi_;  /*< dilatancy angle  */
+    Real alpha_fai_;  /*< Drucker�CPrager��s constants  */
+    Real k_c_;  /*< Drucker�CPrager��s constants */
+public:
```

Comment:
for these member variables, can we directly use term with physical meaning, for exmaple cohesion_ other than c_, for clarity?

Reply:
No problem.

Reply:
https://github.com/Xiangyu-Hu/SPHinXsys/blob/17dc2a2031b947983a12a08c7f22d04609a3bf84/src/shared/materials/elastic_solid.h#L47-L53

I think this is the rule we defined for variables before - using the phonetic pronunciation of the variable instead of its physical meaning, while the physical meaning of the variable is placed inside the constructor function.

I guess this is for the convenience of future calls.